.circleci/config.yml

```diff
@@ -222,7 +222,7 @@ workflows:
             branches:
               ignore: /.*/
             tags:
-              only: /^v[0-9].[0-9].[0-9]+.*/
+              only: /^v\d+\.\d+\.\d+/
```

Comment:
Use same regexp as in core `/^v([0-9])+.([0-9])+.([0-9])+.*/`?

Reply:
But that one can match stuff like `v11a22b33ccc` or `v123456`. Doesn't seems like what we need.

Reply:
You mean because the `.` is not escaped? Then it should be. But mostly was curious why `\d` instead of explicitly digits?

Reply:
Yes, `.` not escaped. `\d` is just shorter, and should be well known. 
Two other concerns with `/^v([0-9])+.([0-9])+.([0-9])+.*/`:
- why we would need matching groups `()` there?
- why _match everything_ tail `.*` needed?

If you think the explicit range is better than `\d`, what about `/^v[0-9]+\.[0-9]+\.[0-9]+/`?

Once we agreed here, I can update core.

Reply:
> why match everything tail .* needed?

to match suffix as `-rc1` etc.

Reply:
so `/^v[0-9]+\.[0-9]+\.[0-9]+.*/` then?

Reply:
updated